api/pkg/app/app.go

```diff
@@ -0,0 +1,196 @@
+package app
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/jinzhu/gorm"
+	"github.com/joho/godotenv"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	// Blank for package side effect: loads postgres drivers
+	_ "github.com/lib/pq"
+)
+
+type Base interface {
```

Comment:
Could we merge the Base and the Config until we have a good reason to split the interfaces (like the experimental/hub/api) ?